packages/google_maps_flutter/google_maps_flutter_web/README.md

```diff
@@ -26,6 +26,8 @@ Modify the `<head>` tag of your `web/index.html` to load the Google Maps JavaScr
 
 Now you should be able to use the Google Maps plugin normally.
 
+To use heatmaps, add `&libraries=visualization` to the end of the URL.
```

Comment:
This is a very good addition to the docs! And something that people currently need to do if they want to draw stuff (`drawing`). Can you add a link to this as well?

* https://developers.google.com/maps/documentation/javascript/libraries

Reply:
Done